pkg/controller/attach_detach.go

```diff
@@ -743,25 +749,13 @@ func NewAttachDetachController(mgr manager.Manager, cloudDiskAttacher CloudDiskA
 }
 
 func (c *SharedState) waitForVolumeAttachmentName(ctx context.Context, azVolumeAttachment *azdiskv1beta2.AzVolumeAttachment) (string, error) {
```

Comment:
It would be good to add a function comment stating that this function waits for the `VolumeAttachment` name to appear in the `azVolumeAttachmentVaMap` which is updated by the `volumeattachment` controller.